build-lib/lib.js

```diff
@@ -39,43 +39,37 @@ module.exports = function (pathConfig, env) {
     footer: `
 window.ShopifyBuy = require('shopify-buy/shopify').default;
 })();`,
-    outputFile: `${pkg.name}.globals.js`,
-    sourceMapConfig: { enabled: false }
+    outputFile: `${pkg.name}.globals.js`
   });
 
   if (env === 'production') {
     const amdOutput = concat(amdTree, {
       inputFiles: ['**/*.js'],
-      outputFile: `${pkg.name}.amd.js`,
-      sourceMapConfig: { enabled: false }
+      outputFile: `${pkg.name}.amd.js`
     });
 
     const polyFilledAmdOutput = concat(mergeTrees([amdOutput, polyfillTree]), {
       headerFiles: ['polyfills.js'],
-      inputFiles: `${pkg.name}.amd.js`,
-      outputFile: `${pkg.name}.polyfilled.amd.js`,
-      sourceMapConfig: { enabled: false }
+      inputFiles: ['**/*.js'],
```

Comment:
No, it doesn't duplicate. `amdOutput` will contain only one output file which is grouped along with `polyfillTree`